tests/ut/backends/test_redis.py

```diff
@@ -1,6 +1,9 @@
 from unittest.mock import ANY, AsyncMock, create_autospec, patch
 
 import pytest
+
+pytest.importorskip("redis")
```

Comment:
I think this approach might be OK (maybe add a flake8 ignore to .flake8 for all of tests/ for the import ordering).